FusionIIIT/applications/globals/urls.py

```diff
@@ -22,4 +22,5 @@
     url(r'^logout/$', views.logout_view, name="logout_view"),
     # API urls
     url(r'^api/auth/', include('applications.globals.api.urls')),
+    url(r'^api/', include('applications.globals.api.dashboardurls')),
```

Comment:
Instead of adding a new entry here, just remove the `auth` from first URL and put the auth in `api/urls.py` file like you've put dashboard.